osu.Game/Localisation/GameplayMenuOverlayStrings.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class GameplayMenuOverlayStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.GameplayMenuOverlay";
+
+        /// <summary>
+        /// "Continue"
+        /// </summary>
+        public static LocalisableString Continue => new TranslatableString(getKey(@"continue"), @"Continue");
+
+        /// <summary>
+        /// "Retry"
+        /// </summary>
+        public static LocalisableString Retry => new TranslatableString(getKey(@"retry"), @"Retry");
+
+        /// <summary>
+        /// "Quit"
+        /// </summary>
+        public static LocalisableString Quit => new TranslatableString(getKey(@"quit"), @"Quit");
```

Comment:
Not sure about these being here rather than common strings. Maybe better to start off with them being here and commonise as necessary?

Reply:
yes,  i'm really not sure this is well placed at all, i considered putting it in common but i was afraid that could be cluttering it